cmd/login/cmd.go

```diff
@@ -50,6 +50,8 @@ const oauthClientId = "ocm-cli"
 
 var reAttempt bool
 
+var env string
```

Comment:
If I'm not mistaken we are slowly trying to move away from "global" variables, would it be possible to rely on args.env or to use exclusively a local variable?

Reply:
We need constantly updated env variable between the run() and CheckAndLogIntoFedramp() methods. We can try and return env from the latter and update in run() but that would just confuse us more. I'd prefer keeping the global variable in this case.